elastic/integrations
# [Cloud Posture] Findings datastream mapping

## What does this PR do?

In the following PR we've adjusted the findings mappings.
- Turned off dynamic mappings
- Imported ECS mappings (auto-generated)
- Aligned mappings to ECS conventions ([reference](https://www.elastic.co/guide/en/ecs/8.2/ecs-conventions.html#_ecs_convention_for_indexing_text_fields))

This means that from now on we opt-in which fields we index in the findings being sent by `cloudbeat`



## Checklist


## Author's Checklist



## How to test this PR locally


To test this PR I recommend building the package locally and use it, send data using `cloudbeat` and check that all fields have the correct mappings

## Related issues


-

## Screenshots
